README.md

```diff
@@ -33,7 +31,7 @@ Add the following lines to your `composer.json` file:
 
 ## Developing Coding Standards
 
-Coding standard can be developed in the way [PHP_CodeSniffer] documents [this].
+Coding standard can be developed in the way [PHP_CodeSniffer][codesniffer] documents [this][tutorial].
```

Comment:
Please avoid using "this" for links. Suggestion:

    Coding standard can be developed in the way [PHP_CodeSniffer][codesniffer] documents in their [tutorial page][tutorial].